js/apps/admin-ui/src/identity-providers/add/AdvancedSettings.tsx

```diff
@@ -125,6 +140,89 @@ export const AdvancedSettings = ({ isOIDC, isSAML }: AdvancedSettingsProps) => {
       />
       <SwitchField field="config.hideOnLoginPage" label="hideOnLoginPage" />
 
+      {(!isSAML || isOIDC) && (
+        <FormGroupField label="filteredByClaim">
+          <Controller
+            name="config.filteredByClaim"
+            defaultValue="false"
+            control={control}
+            render={({ field }) => (
+              <Switch
+                id="filteredByClaim"
+                label={t("common:on")}
+                labelOff={t("common:off")}
+                isChecked={field.value === "true"}
+                onChange={(value) => {
+                  field.onChange(value.toString());
+                }}
+                aria-label="filteredByClaim"
```

Comment:
This can be removed as the field is already labeled through `FormGroupField`.

Reply:
Removing "aria-label" field thanks